libs/ui/src/lib/header/header.component.scss

```diff
@@ -0,0 +1,3 @@
+.flexExpand {
+  flex: 1 1 auto;
+}
```

Comment:
You can delete this and instead of putting a span, telling to the toolbar to take more space.
Look at the footer css, you will get an example, and maybe you can do something like
```
mat-toolbar-row {
  display: flex;
  justify-content: space-between;
```
Also, you can use the flexLayout, I guess it works on the toolbar.

Reply:
Made changes according to this and it works.